foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -171,7 +171,14 @@ private static void duplicateServiceCombConfigToCse(AbstractConfiguration source
       }
 
       String cseKey = CONFIG_CSE_PREFIX + key.substring(key.indexOf(".") + 1);
-      source.addProperty(cseKey, source.getProperty(key));
+      if (!source.containsKey(cseKey)) {
+        source.addProperty(cseKey, source.getProperty(key));
+      } else {
+        LOGGER
+            .warn(
```

Comment:
Not sure if the warning message is enough for this time, if the configuration is wrong, java-chassis may need to stop working instead of sending out the warning message to let the admin know about it.
BTW,  we may need a default override policy here.

Reply:
As shown in test case, this can work. Just when merge, it will not take the cse-config-order(which is implemented by config-cc). We can not cover all orders for ConfigurationsImpl, so give a warning  message. 
If we cover this, we need to change any implementations. This is not a good idea.

Reply:
"stop working" can not be applied because we may configure some default values using cse or servicecomb, if doing this users code have to change to work properly. We can't do this now.